src/core/jsonschema/official_resolver.in.cc

```diff
@@ -52,21 +56,16 @@ auto sourcemeta::core::schema_official_resolver(std::string_view identifier)
     return sourcemeta::core::parse_json(
         R"EOF(@METASCHEMA_JSONSCHEMA_2020_12_OUTPUT@)EOF");
 
-    // Just for compatibility given that this is such a common issue
-  } else if (identifier == "https://json-schema.org/draft/2020-12/schema#") {
-    return sourcemeta::core::parse_json(
-        R"EOF(@METASCHEMA_JSONSCHEMA_2020_12@)EOF");
-  } else if (identifier ==
-             "https://json-schema.org/draft/2020-12/hyper-schema#") {
-    return sourcemeta::core::parse_json(
-        R"EOF(@METASCHEMA_HYPERSCHEMA_2020_12@)EOF");
-
     // JSON Schema 2019-09
-  } else if (identifier == "https://json-schema.org/draft/2019-09/schema") {
+  } else if (identifier == "https://json-schema.org/draft/2019-09/schema" ||
+             // Just for compatibility given that this is such a common issue
+             identifier == "https://json-schema.org/draft/2019-09/schema#") {
     return sourcemeta::core::parse_json(
         R"EOF(@METASCHEMA_JSONSCHEMA_2019_09@)EOF");
   } else if (identifier ==
-             "https://json-schema.org/draft/2019-09/hyper-schema") {
+                 "https://json-schema.org/draft/2019-09/hyper-schema" ||
+             identifier ==
```

Comment:
Can you add the comment about compatibility above this line too?

Reply:
done